### DISCUSSION TITLE
posts/first/

### DISCUSSION THREAD

**Discussion by giscus** [22000510/22000510.github.io#7] [General]
# posts/first/

안녕하세요! 이동은이라고 합니다! 고생 끝에 깃허브 블로그를 열게 되었습니다.(어렵더라구요..ㅠㅠ) 구글링을 하다보니 깃허브로 블로그 올리는 법을 정리해놓은 블로그가 많아 많은 도움을 받았습니다. 감사합니다. 휴학하는 동안 공부하는 것들을 정리해서 올려보려고 합니다. 많이 부족한 포스팅이 되겠지만 발전해가는 과정 보여드릴 수 있도록 노력하겠습니다!

https://22000510.github.io/posts/first/

**Comment by 22000510** [comment]
연습 채팅 입니다.
